tests/specs/breadcrumbs/automatic.xhr.js

```diff
@@ -24,8 +25,26 @@ describe("XHR tracking", function() {
     expect(breadcrumbs[2].type).toBe("request");
     expect(breadcrumbs[2].message).toContain("Opening request");
 
```

Comment:
What would the third breadcrumb be in this case?

Reply:
I think it's a different request starting